### DISCUSSION TITLE
Does Wrapper check current Win10 settings at startup and display current state in GUI?

### DISCUSSION THREAD

**Discussion by tobwz** [farag2/Sophia-Script-for-Windows#167] [Q&A] [answered]
Does Sophia Script Wrapper check at startup the current state of the Win10 settings and display it?

It seems to be not.
That would be bad.

Or do I have to prepare something in Wrapper to enable pre-check?

**Comment by farag2** [comment, marked as answer]
No. You should Import a preset file to make it display the preset settings.
